packages/web/src/scss/components/Tooltip/index.html

```diff
@@ -320,102 +414,158 @@ <h2 class="docs-Heading">Dismissible Tooltip via JS API</h2>
             </svg>
             <span class="accessibility-hidden">Close</span>
           </button>
-          <span class="Tooltip__arrow"></span>
+          <span class="Tooltip__arrow" data-spirit-element="arrow"></span>
         </div>
       </div>
 
     </div>
 
   </section>
 
-  <section id="full-floating-ui" class="docs-Section spirit-feature-tooltip-enable-data-placement">
+  <section class="docs-Section spirit-feature-tooltip-enable-data-placement">
 
-    <h2 class="docs-Heading">Full Floating UI Usage</h2>
+    <script type="module" src="floating-ui-example.mjs"></script>
+
+    <style>
+      .example-viewport {
+        width: 100%;
+        max-width: 100%;
+        height: 30rem;
+        overflow: auto;
+      }
+
+      .example-content {
+        position: relative;
+        width: 300%;
+        height: 90rem;
+        padding-block: 44rem;
+        text-align: center;
+      }
+    </style>
+
+    <h2 class="docs-Heading">FloatingUI Playground</h2>
+
+    <p class="mb-1000">
+      Try scrolling the frame or resizing the window to see how the Tooltip behaves. The Floating UI
+      library is trying to keep the Tooltip in the viewport and it is also flipping, shifting and
+      resizing the Tooltip when it is not possible to keep it in the viewport.
+    </p>
 
     <div class="docs-Stack docs-Stack--stretch">
 
-      <p class="mb-0">
-        The following example is using external library <a href="https://floating-ui.com">Floating UI</a>.
-        Try scrolling the frame or resizing the window to see how the Tooltip behaves. The Floating UI
-        library is trying to keep the Tooltip in the viewport and it is also flipping, shifting and
-        resizing the Tooltip when it is not possible to keep it in the viewport.
-      </p>
-
-      <div class="Alert Alert--informative" role="alert">
-        <svg width="24" height="24">
-          <use xlink:href="/icons/svg/sprite.svg#info" />
-        </svg>
+      <div class="Grid Grid--cols-4">
```

Comment:
The playground controls now expand the page horizontally and some scenarios (especially the `size` feature) cannot be shown. I was wondering if you could make the controls responsive, something like:

```suggestion
      <div class="Grid Grid--cols-1 Grid--tablet--cols-4">
```

Similarly for the select boxes below:

```html
      <div class="Grid Grid--cols-1 Grid--tablet--cols-2">
```

![image](https://github.com/lmc-eu/spirit-design-system/assets/5614085/2db09860-c74c-41c1-9059-b245270fdf8b)